src/payment/strategies/cardinal/cardinal-three-d-secure-flow-v2.ts

```diff
@@ -112,4 +112,14 @@ export default class CardinalThreeDSecureFlowV2 {
 
         return bin || '';
     }
+
+    private async _configure(methodId: string, token: string): Promise<void> {
+        if (this._cardinalClient.isConfigured()) {
+            const { config: { testMode } } = this._store.getState().paymentMethods.getPaymentMethodOrThrow(methodId);
+
+            return this._cardinalClient.reconfigure(methodId, token, testMode);
+        }
+
+        return this._cardinalClient.configure(token);
```

Comment:
as per the suggestions above, could this simply be `this._cardinalClient.configure(token);` ? I'm assuming `methodId` and `testMode` don't really need to be re-configured? maybe those things can be stored internally in the cardinalClient?